data/transit/route/ferry.json

```diff
@@ -106,6 +106,17 @@
       }
     },
     {
+      "displayName": "Casco Bay Lines",
+      "id": "cascobaylines",
+      "locationSet": {"include": ["us-me"]},
+      "tags": {
+        "network": "Casco Bay Lines",
+        "network:wikidata": "Q5048264",
+        "operator": "Casco Bay Lines",
+        "operator:wikidata": "Q5048264",
```

Comment:
It seems that Casco Bay Lines is both the network itself and its operator? But we know that a ferry network and a transport company are two different things.
It is possible to have such a tag, but it is indeed a bit strange, and it can be 2 different items if possible.

Reply:
OK, I will try to go into the code and get rid of the network info